packages/inertia/package.json

```diff
@@ -19,7 +19,9 @@
     "watch": "microbundle watch --format cjs"
   },
   "dependencies": {
-    "axios": "^0.19.0 || ^0.20.0"
+    "axios": "^0.19.0 || ^0.20.0",
+    "deepmerge": "^4.2.2",
+    "qs": "^6.9.4"
```

Comment:
Basically my goal here being to improve the chances that multiple packages can share the same version of these two packages, since they are both very popular and used in other packages a lot. I'm just not sure how low to go with the versions.

Reply:
Yeah, I think that's fine.